mlflow/server/js/src/model-registry/components/ModelVersionView.js

```diff
@@ -490,7 +497,7 @@ const mapStateToProps = (state, ownProps) => {
   const { modelName } = ownProps;
   const { version } = ownProps.modelVersion;
   const tags = getModelVersionTags(modelName, version, state);
-  return { tags };
+  return { tags, modelSource: ownProps.modelVersion.source };
```

Comment:
not sure if it's possible but let's make make it defensive against dereferencing exceptions
```suggestion
  return { tags, modelSource: ownProps.modelVersion?.source };
```